frontend/components/FieldMedia/FieldMediaEdit.jsx

```diff
@@ -137,103 +137,131 @@ export default class FieldImageEdit extends Component {
     const singleFile = schema.settings && schema.settings.limit === 1
     const values = (value && !Array.isArray(value)) ? [value] : value
 
+
     return (
-      <Label label={displayName}>
+      <Label label={displayName} className={styles.label}>
         {values &&
-          <div class={styles['value-container']}>
-            <div class={styles.thumbnails}>
-              {values.map(value => (
-                <img
-                  class={styles.thumbnail}
-                  src={this.getImageSrc(value)}
+          values.map((value, idx) => {
+
+            let file = {}
+            file.name = value.fileName.split('.').slice(0, -1).join('.')
+            file.size = fileSize(value.contentLength).human('si') || ''
+            file.ext  = value.fileName.split('.').pop()
+
+            let styleValueContainer = new Style(styles, 'value-container')
+              .addIf('value-container-first', values.length > 1 && idx === 0)
```

Comment:
Instead of styling based on classes that are added if an element is the first, the last or the only one, could we not do it using CSS only using `:first-child`, `:last-child` and `:first-child:last-child`?

Reply:
I tried those (also in our case of the mocku pagination button), but `:first-child` seem to have no effect. I think they are only for targeting given element _not_ elements by class name.

Reply:
👍

Reply:
That's correct. `:first-child` will target the first element amongst its siblings, as shown by the screenshot:

<img width="1224" alt="screenshot 2018-10-24 at 15 57 12" src="https://user-images.githubusercontent.com/4162329/47440189-92f67080-d7a5-11e8-9592-b18e5842d24a.png">

The problem here is that the items we're looking to target aren't the only siblings under the same parent, so `:last-child` will target the wrong element.

You could solve it by creating a `div` to group only the relevant items and that way make use of `:first-child` and `:last-child`. I would actually prefer it this way, but I'm not fussed. Happy for it to be left as is!